src/aura/gatherer/ComponentSerializer.js

```diff
@@ -384,3 +378,21 @@ function getBody(component) {
         expressions: bodyMapExpressions
     };
 }
+
+function getAttributeExpression(component, attribute) {
+    let expression = null;
+    try {
+        expression = $A.componentService.getAttributeExpression(component, attribute);
+    } catch (e) {
+        // getAttributeExpression() in 222 does not support interop component
+        // lucily the attributes property is not minified, giving us
+        // the ability to monkey patch it.
+        // In 224 getAttributeExpression() should support interop and we can remove this.
+        if (component && component.attributes) {
+            if ($A.util.isExpression(component.attributes[attribute])) {
```

Comment:
assuming $A.util.isExpression can handle undefined, if the attribute is not present